modules/core/server/views/layout.server.view.html

```diff
@@ -26,14 +26,14 @@
 	<meta property="og:title" content="{{title}}">
 	<meta property="og:description" content="{{description}}">
 	<meta property="og:url" content="{{url}}">
-	<meta property="og:image" content="/img/brand/logo.png">
+	<meta property="og:image" content="{{url}}modules/core/img/brand/logo.png">
 	<meta property="og:type" content="website">
 
 	<!-- Twitter META -->
 	<meta name="twitter:title" content="{{title}}">
 	<meta name="twitter:description" content="{{description}}">
 	<meta name="twitter:url" content="{{url}}">
-	<meta name="twitter:image" content="/img/brand/logo.png">
+	<meta name="twitter:image" content="{{url}}modules/core/img/brand/logo.png">
```

Comment:
@ilanbiala You need to full URL to satisfy twitter and facebook. See @simison's comment at https://github.com/meanjs/mean/pull/322